src/ziggurat/messaging/producer.clj

```diff
@@ -94,9 +94,20 @@
 (defn- get-channel-retry-count [topic-entity channel]
```

Comment:
changes similar to the ones mentioned in the PR against master

Reply:
@theanirudhvyas updated with same changes against master